client/js/constants/constants.js

```diff
@@ -4,12 +4,14 @@ var AppConstants = {
   ViewActionTypes: keyMirror({
     ADD_EVENT: null,
     GET_DAY_DATA: null,
-    GET_MONTH_DATA: null
+    GET_MONTH_DATA: null,
+    STAFF_MOVED: null
   }),
   ServerActionTypes: keyMirror({
     EVENT_ADDED: null,
     DAY_DATA_RECIEVED: null,
-    FETCHED_MONTH_DATA: null
+    FETCHED_MONTH_DATA: null,
+    STAFF_MOVED: null
```

Comment:
The serverActionType is not currently used - after the data is updated on the backend, the ViewActionCreator.getDayData action is triggered, which sends a DAY_DATA_RECEIVED actionType. I should remove it. 

At some point, it may be beneficial to add an EmployeeStore to the flux architecture.